Formula/libsql-server.rb

```diff
@@ -0,0 +1,28 @@
+class LibsqlServer < Formula
+  desc "Server-mode binary for libSQL (sqld), an Open Contribution fork of SQLite"
+  homepage "https://libsql.org"
+  url "https://github.com/libsql/sqld/archive/refs/tags/v0.9.0.tar.gz"
+  sha256 "d3b222ed54c4ab1707da16198760f8619d5cdbfbc6ecd72998671100ddc85439"
+  license "MIT"
+
+  depends_on "llvm" => :build
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "openssl@3"
```

Comment:
Actually it looks like there is linkage only on Linux. Does it use a different SSL implementation on macOS? Or did it build its own copy of OpenSSL to statically link to?